packages/react/src/lib/config/supportedWallets.ts

```diff
@@ -0,0 +1,85 @@
+import { SeiWallet } from '../provider';
+
+export const FIN_WALLET: SeiWallet = {
+	getAccounts: async (chainId) => {
+		const offlineSigner = await window?.['fin']?.getOfflineSigner(chainId);
```

Comment:
we should actually replace these `getOfflineSigner` with `getOfflineSignerAuto` to support ledgers

Reply:
Updated